cdc/entry/mounter.go

```diff
@@ -173,7 +172,23 @@ func (m *mounterImpl) unmarshalAndMountRowChanged(ctx context.Context, raw *mode
 			if rowKV == nil {
 				return nil, nil
 			}
-			return m.mountRowKVEntry(tableInfo, rowKV, raw.ApproximateDataSize())
+			row, err := m.mountRowKVEntry(tableInfo, rowKV, raw.ApproximateDataSize())
```

Comment:
How about the following? So we don't need to change model.
```suggestion
			row, datum, err := m.mountRowKVEntry(tableInfo, rowKV, raw.ApproximateDataSize())
```